### PR TITLE
Email Composition

### PR DESCRIPTION
**Pull request by jameswatters1**
Added a button for Create Email on the Inbox page. It opens the CreateEmail component and allows you to send an email to 1 or multiple recipients, optionally CC email address, write a subject line, add a file (although it is not functional yet), and type out a body in the email. You can then send or cancel the email clearing the form. After sending the email, the CreateEmail component stays open, but I figured we can fix this when we change our overall navigation.

### CODE REVIEW

**Comment by D3ISM3** [comment]
So you tested this and it actually sends out an email? @jameswatters1

**Comment by jameswatters1** [comment]
Yes it worked for me. Did you try it and it didn't work? I was having problems originally and I had to restart the application and make sure when i signed in that I gave the application and google quickstart permission to send emails.

**Comment by D3ISM3** [comment]
No, it's fine! I just wanted to verify. I had sent that when I was viewing github on my phone so I couldn't readily test it myself.

Things look good to me!

File: ec_email_client/src/index.js [hunk 1]
```diff
@@ -15,3 +15,7 @@ ReactDOM.render(
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
 serviceWorker.unregister();
+
+
+
+
```
Comment (D3ISM3) [inline ec_email_client/src/index.js:21]: I too believe that whitespace are important for readability 😎

**Review by D3ISM3 — APPROVED** [review]

**Comment by pbell97** [comment]
Look great. Just tested and it works flawlessly.

**Review by pbell97 — APPROVED** [review]
Works great! Just tested and works flawlessly.

**Pull request merged**